mat64/solve.go

```diff
@@ -11,15 +11,15 @@ import (
 	"github.com/gonum/matrix"
 )
 
-// Solve solves a minimum-norm solution to a system of linear equations defined
-// by the matrices a and b. If a is singular or near-singular a Condition error
+// Solve finds a minimum-norm solution to a system of linear equations defined
+// by the matrices A and b. If A is singular or near-singular a Condition error
```

Comment:
This was intentional. Though would could change the decision. `a` is the name of the variable in the signature while A is the convention for the matrix notation. These are in conflict.